Added tutorials/guides for QGIS and postgres

vermeulendivan opened the pull request:
New section(s) added to the handbook, with focus on tutorials, guides and hints.
This includes a section for qgis and postgres. The qgis section focuses on the Atlas and issues a user can encounter. The postgres section focuses how to work with postgres in qgis. The postgres section covers postgis, restoration, data loading, storage in a db of spatial data and qgis projects, etc. 

Atlas:
![image](https://user-images.githubusercontent.com/79740955/231223776-a7505af5-51e2-49db-85d6-8bb6bc3f5f26.png)

QGIS issues:
![image](https://user-images.githubusercontent.com/79740955/231223970-3467b708-a8c1-4b31-8302-f52647b26467.png)

QGIS and postgres:
![image](https://user-images.githubusercontent.com/79740955/231224100-10d3c394-5ab1-4904-a571-7e6fb0652606.png)

Postgres service file:
![image](https://user-images.githubusercontent.com/79740955/231224257-c7d670bb-874a-4a3c-b5a0-028a828e4cb4.png)

Postgres issues:
![image](https://user-images.githubusercontent.com/79740955/231224399-912265d2-631b-4759-8345-b093dff0c11b.png)

timlinux commented:
@vermeulendivan since these are GIS related resources, can you rather file them here https://github.com/kartoza/TheKartozaHandbook/tree/main/docs/src/gis, improving organisation as you go if needed.....

Love seeing this work coming in, thanks!

vermeulendivan commented:
@timlinux Thanks for the feedback! Had a feeling you would like me to move it around (were unsure where the best spot will be), so thought its best to ask for a review from your side. I would let you know when I've moved it, likely sometime this weekend.

vermeulendivan commented:
@timlinux Made the suggested changes, and moved the GIS resources. Also made minor improvements to the content. The GIS resources will now be up to date. Pretty sure there can come improvements, and additional sections to the resources.

Added a resources section under development, which now contains the google cloud platform guide:

![image](https://user-images.githubusercontent.com/79740955/232315051-03834da6-d354-48c2-8e59-cc343e9f480c.png)

![image](https://user-images.githubusercontent.com/79740955/232315074-29195eed-d591-43e6-bec8-6ed6a982451c.png)

![image](https://user-images.githubusercontent.com/79740955/232315080-9d48dc62-6846-410e-8b00-8ca248b4334d.png)

And much more.

There are two more sections I need to finish for the GCP section:
- Pub/Sub trigger events
- Dataflow pipeline using Python
